src/ParquetMsg.chpl

```diff
@@ -484,9 +484,9 @@ module ParquetMsg {
       }
     } else if mode == TRUNCATE {
       if matchingFilenames.size > 0 {
-        warnFlag = true;
+        filesExist = true;
       } else {
-        warnFlag = false;
+        filesExist = false;
       }
```

Comment:
Since we are now defaulting `filesExist` to true, we could simplify this block to 

```chapel
if matchingFilenames.size == 0 {
    filesExist = false;
}
```